addon/components/multi-faceted-search/facet.js

```diff
@@ -5,9 +5,11 @@ let Facet = Ember.Object.extend({
   type: "", // "checkbox" or "link"
   terms: [], // each term: { value: "", display: string.optional, count: num.optional, selected: boolean }
   emptyMessage: "Nothing to display",
+  hasError: false,
 
   checkbox: Ember.computed.equal('type', 'checkbox'),
   link: Ember.computed.equal('type', 'link'),
+  datetime: Ember.computed.equal('type', 'datetime'),
```

Comment:
They appear the same way as any other facet. For example: `Time: Start`, `Time: End`, `Time: 2 Selected`; the chosen date-time from the picker is the value of the term, thus not displayed in the "Your Selections: " bar. Every time the date-time is clicked, it's selected / unselected status toggles.